content/src/editor.less

```diff
@@ -805,19 +818,34 @@ body#pencildoc #diigolet-csm {
 }
 
 .directory .item {
+  text-align: center;
   display: inline-block;
   float: left;
   clear: left;
   text-decoration: none;
-  padding: 0 28px 0 12px;
-  min-height: 28px;
   color: #222;
-  cursor: pointer
+  cursor: pointer;
+  div {
+    width: 200px;
```

Comment:
Recommend a denser layout so that students get 3-4 columns rather than 2-3 columns in their listings on a laptop.

```css
div {
  width: 159px
  height: 159px
  # etc...
}
.thumbnail {
  width: 100px
  height: 100px
  margin-top: 16px;
  border: 1px solid #ccc;
  border-radius: 4px;
  background-color: #eee;
  padding: 1px;
}
```

And recommend a reactive border instead of always blue:

```css
.thumbnail:hover {
  padding: 0;
  border-width: 2px;
  border-color: blue
}
```